nav2_smart_planner/src/smart_planner.cpp

```diff
@@ -269,6 +269,25 @@ SmartPlanner::makePlan(
   return !plan.poses.empty();
 }
 
+void
+SmartPlanner::smoothApproachToGoal(
+  const geometry_msgs::msg::Pose & best_pose,
```

Comment:
In the header file the Pose is called "goal", here you're calling it "best_pose". I'm confused? I think goal_pose is the most clear

Reply:
I'll fix this.